2-ui/4-forms-controls/4-forms-submit/article.md

```diff
@@ -1,25 +1,25 @@
-# Forms: event and method submit
```

Comment:
Formalar qolgani maqul meni fikrimcha, sababi so'rovnomalarni forma ham deymizku to'g'rimi?